shell/src/main/java/org/crsh/shell/impl/command/system/help.java

```diff
@@ -75,12 +75,7 @@ public void main(InvocationContext<Object> context) throws Exception {
       }
     }
 
-    String safeStr = safeShell ? "SAFE-" : "UNSAFE-";
-    String sshStr = sshMode ? "SSH-" : "";
-    String saStr = standAlone ? "Standalone-" : "";
-    String intStr = internal ? "Internal-" : "";
-    String pref = "[" + safeStr + saStr + intStr + sshStr + "Shell]: ";
-    context.provide(new LabelElement(pref + "Try one of these commands with the -h or --help switch:\n"));
+    context.provide(new LabelElement("Try one of these commands with the -h or --help switch:\n"));
```

Comment:
Please add back in the original information regarding what mode the shell is in: The 'pref' variable added to the start of the string in the label element.

Reply:
Done